src/Swashbuckle.AspNetCore.Newtonsoft/SchemaGenerator/NewtonsoftDataContractResolver.cs

```diff
@@ -143,7 +143,7 @@ private string JsonConverterFunc(object value)
             return JsonConvert.SerializeObject(value, _serializerSettings);
         }
 
-        private IEnumerable<DataProperty> GetDataPropertiesFor(JsonObjectContract jsonObjectContract, out Type extensionDataType)
```

Comment:
What's the rationale for this change? As a general rule, I try to maintain the least specific method signature that's required

Reply:
These resolve suggestions from the [CA1859](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1859) analyzer to use specific types in private/internal implementations to get better performance from the JIT by avoiding virtual dispatch. Visual Studio pointed these out to my while I was looking through files while fixing the issue.

Reply:
OK cool thx for the explanation 👍